discovery_engine/test/feed_manager_test.dart

```diff
@@ -60,145 +60,149 @@ Future<void> main() async {
-      expect(activeRepo.box, isEmpty);
-
-      // id2 should now be deactivated, id3 still inactive
-      expect(docRepo.box, hasLength(2));
-      final docs = await docRepo.fetchByIds({id2, id3});
-      expect(docs, hasLength(2));
-      expect(docs[0].isActive, isFalse);
-      expect(docs[1].isActive, isFalse);
-    });
-
-    test('get next feed batch', () async {
-      final evt = await mgr.nextFeedBatch();
-      expect(engine.getCallCount('getFeedDocuments'), equals(1));
-      final docs =
-          evt.whenOrNull(nextFeedBatchRequestSucceeded: (docs) => docs);
-
-      // check returned ids match those of mock engine
-      expect(docs, isNotNull);
-      expect(
-        docs!.map((doc) => doc.documentId),
-        unorderedEquals(
-          engine.feedDocuments
-              .map<DocumentId>((doc) => doc.document.documentId),
-        ),
-      );
-
-      // check repositories are updated
-      expect(docRepo.box, hasLength(4));
-      expect(
-        docRepo.box.values,
-        containsAll(engine.feedDocuments.map<Document>((doc) => doc.document)),
-      );
-      expect(activeRepo.box, hasLength(3));
-      expect(
-        activeRepo.box.values,
-        containsAll(
-          engine.feedDocuments.map<ActiveDocumentData>((doc) => doc.data),
-        ),
-      );
-
-      // serialize should be called and state saved
-      expect(engine.getCallCount('serialize'), equals(1));
-      expect(engineStateRepo.box.isNotEmpty, isTrue);
-    });
-
-    test('restore feed', () async {
-      final earlier = DateTime.utc(1969, 7, 20);
-      final later = DateTime.utc(1989, 11, 9);
-      // ignore: deprecated_member_use_from_same_package
-      await docRepo.update(doc2..timestamp = earlier);
-      // ignore: deprecated_member_use_from_same_package
-      await docRepo.update(doc3..timestamp = later);
-      // ignore: deprecated_member_use_from_same_package
-      await docRepo.update(engine.feedDocuments[0].document..timestamp = later);
-      await docRepo
-          // ignore: deprecated_member_use_from_same_package
-          .update(engine.feedDocuments[1].document..timestamp = earlier);
-
-      expect(docRepo.box, hasLength(4));
-
-      final evt = await mgr.restoreFeed();
-      final feed = evt.whenOrNull(restoreFeedSucceeded: (docs) => docs);
-
-      expect(feed, isNotNull);
-      expect(feed, hasLength(3));
-      // doc1, doc2 have the earlier timestamp
-      expect(
-        feed![0].documentId,
-        equals(engine.feedDocuments[1].document.documentId),
-      );
-      expect(feed[1].documentId, equals(doc2.documentId));
-      // doc0 has the later timestamp
-      expect(
-        feed[2].documentId,
-        equals(engine.feedDocuments[0].document.documentId),
-      );
-      // doc3 is excluded since it is inactive
-    });
-    //TODO[pmk] test use mock engine, check which tests can be recycled
-    //ignore:require_trailing_commas
-  }, skip: cfgFeatureStorage);
+  group(
```

Comment:
only indention changed due to how `skip:` + `require_trailing_comas` interacts badly